app/models/orchestration_template_cfn.rb

```diff
@@ -30,6 +30,16 @@ def parameters
     end
   end
 
+  def resources
+    raw_resources = JSON.parse(content)["Resources"]
```

Comment:
I think this should be

```
@resources ||= (JSON.parse(content)["Resources"] || {}).collect do |key, val|
    ... init resources
  end
end
```

That way it's cached in an ivar only initialized one time.  The resources shouldn't change frequently enough to cause any problems with this.